clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/BaseGoogleInstanceDescription.groovy

```diff
@@ -43,8 +43,18 @@ class BaseGoogleInstanceDescription extends AbstractGoogleCredentialsDescription
   Boolean automaticRestart
   OnHostMaintenance onHostMaintenance
 
+  // We support passing the image to deploy as either a string or an artifact, but default to
+  // the string for backwards-compatibility
```

Comment:
What is the difference between STRING and ARTIFACT in this context?

Reply:
It tells us whether to pull the image to deploy from `image` (`STRING`) or `imageArtifact` (`ARTIFACT`)